schemas/layer-3.cue

```diff
@@ -0,0 +1,67 @@
+// layer-3-policy.cue
+
+// TODO
+// Create schema for a policy document
+// Policy docs should reference common and unique risk definitions, so we need schema for that as well
+
+#Policy: {
+    // Metadata useful for evaluation and automation
+    metadata?: {
+        version: string
+        author: string
+        last_modified: string
+        sci_version: string
+        remarks: string
+    }
+
+    // Unique identifier for this policy
+    id: string
+
+    // Human-readable title of the policy
+    title: string
+
+    // Short description of this policy’s intent or purpose
+    description: string
+
+    // Optional reference to a parent policy this one inherits from or refines
+    parent_policy_id?: string
+
+    // Policy classification level (e.g., mandatory, recommended)
+    classification: oneOf("mandatory", "recommended")
+
+    // Reference to one or more Layer 2 control catalogs
+    control_catalogs: [...#CatalogReference]
+
+}
+
+#CatalogReference: {
+
+    id: string
+
+    version: string
+
+    // List of IDs to applicability values defined in the catalog
```

Comment:
I think this points to a need to better document/define applicability as a concept. I'm not following

Reply:
@eddie-knight As I dove further into my branch with generated types and tried to implement Evaluate, I ran into a perfect use case for Layer 3 to bridge between 2 and 4, see https://github.com/trumant/sci/pull/1/files#diff-9902cbfa7faa6b2d39761d5085851aa84657f1acf16dcc400480ba4bcbaa59d4R14-R22

Curious to hear your view on where you see the filtering occurring

Reply:
https://github.com/trumant/sci/pull/1/files#diff-f516c03e4831eac9448107db61b07347cdba8023233c85638cb207faadded288 also shows where policy could play a role